doc/data/messages/a/arguments-renamed/bad.py

```diff
@@ -0,0 +1,8 @@
+class Fruit:
+    def brew(self, fruit_name: str):
+        print(f"Brewing a fruit named {fruit_name}")
+
+
+class Orange(Fruit):
+    def brew(self, orange_name: str):  # [arguments-renamed]
+        print(f"Brewing an orange named {orange_name}")
```

Comment:
```suggestion
class Fruit:
    def brew(self, ingredient_name: str):
        print(f"Brewing a {type(self)} with {ingredient_name}")

class Apple(Fruit):
    ...

class Orange(Fruit):
    def brew(self, flavor: str):  # [arguments-renamed]
        print(f"Brewing an orange with {flavor}")
        
for fruit, ingredient_name in [[Orange(), "thyme"], [Apple(), "cinnamon"]]:
    fruit.brew(ingredient_name=ingredient_name)
```
Adding an example of why it's wrong to rename an argument.